src/pages/Bridge/index.tsx

```diff
@@ -0,0 +1,242 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import BigNumber from 'bignumber.js';
+import { useCallback, useMemo } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import {
+  ApproveTokenSteps,
+  Card,
+  Icon,
+  LabeledInlineContent,
+  PrimaryButton,
+  SpendingLimit,
+  TextButton,
+  TokenTextField,
+} from 'components';
+import { Link } from 'containers/Link';
+import useFormatTokensToReadableValue from 'hooks/useFormatTokensToReadableValue';
+import useTokenApproval from 'hooks/useTokenApproval';
+import { useGetToken } from 'packages/tokens';
+import { useTranslation } from 'packages/translations';
+import { chains, useAccountAddress, useAuthModal, useChainId } from 'packages/wallet';
+
+import { ChainSelect } from './ChainSelect';
+import { ReactComponent as LayerZeroLogo } from './layerZeroLogo.svg';
+import TEST_IDS from './testIds';
+
+const formSchema = z.object({
+  fromChainId: z.number(),
+  toChainId: z.number(),
+  amountTokens: z.string().min(1),
+});
+
+const BRIDGE_DOC_URL = ''; // TODO: add link (see VEN-2248)
+
+const BridgePage: React.FC = () => {
+  const { t } = useTranslation();
+  const bridgeContractAddress = ''; // TODO: get address from bridge contract (see VEN-2248)
+  const { chainId } = useChainId();
+  const { openAuthModal } = useAuthModal();
+  const { accountAddress } = useAccountAddress();
+  const isUserConnected = !!accountAddress;
+
+  const { control, handleSubmit, formState, watch, getValues, setValue } = useForm<
+    z.infer<typeof formSchema>
+  >({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      fromChainId: chainId,
+      toChainId: chains.find(chain => chain.id !== chainId)?.id,
+      amountTokens: '',
+    },
+  });
+
+  const fromChainId = watch('fromChainId');
+
+  const xvs = useGetToken({
+    symbol: 'XVS',
+    chainId: fromChainId,
+  });
+
+  const {
+    isTokenApproved: isXvsApproved,
+    approveToken: approveXvs,
+    isApproveTokenLoading: isApproveXvsLoading,
+    isWalletSpendingLimitLoading: isXvsWalletSpendingLimitLoading,
+    walletSpendingLimitTokens: xvsWalletSpendingLimitTokens,
+    revokeWalletSpendingLimit: revokeXvsWalletSpendingLimit,
+    isRevokeWalletSpendingLimitLoading: isRevokeXvsWalletSpendingLimitLoading,
+  } = useTokenApproval({
+    token: xvs!,
+    spenderAddress: bridgeContractAddress,
+    accountAddress,
+  });
+
+  const switchChains = useCallback(() => {
+    const formValues = getValues();
+    setValue('fromChainId', formValues.toChainId);
+    setValue('toChainId', formValues.fromChainId);
+  }, [getValues, setValue]);
+
+  const onSubmit = (values: z.infer<typeof formSchema>) => {
+    // TODO: send transaction (see VEN-2248)
+    console.log(values);
+  };
+
+  // TODO: wire up
+  const walletBalanceTokens = new BigNumber(10);
```

Comment:
@therealemjy I've a question, I'm wondering how this will work if the user changes the "from" chain. Shouldn't "from" be locked to the chain you are connected to?

Reply:
Unless we can get the balance the user has on each chain no matter which one they are connected to

Reply:
Yeah that's a good point. Technically we can fetch the balance on each chain (the app is setup so a provider is given for each chain).
However I think you're right and we should still lock the "from" chain, as users won't be able to send a transaction anyway if the from chain doesn't correspond to the one their wallet is connected to. I'll discuss this with Jesús and Pavel.

Reply:
I spoke with Jesús and Pavel and the decision taken is to link the "from" chain select to the wallet chain. This means changing the chain will also change the wallet chain, and vice versa.